### PR TITLE
OKTA-86328

### PR DESCRIPTION
**Pull request by larsjohansen-okta**
- Updated AMR values for OIDC according to the newest spec: https://tools.ietf.org/html/draft-jones-oauth-amr-values-05

Primary reviewer: @tthompson-okta 
Secondary reviewers: @federations-okta @mystiberry-okta @karlmcguinness-okta

### CODE REVIEW

**Comment by mystiberry-okta** [comment]
@larsjohansen-okta looks great to me. Will merge for next week's preview release, yes?

**Comment by larsjohansen-okta** [comment]
If @tthompson-okta signs off on this before CF on Friday, I'll merge the corresponding code https://github.com/okta/okta-core/pull/13704 and then we'll merge this for next week's preview release.

File: _source/docs/api/resources/sessions.md [hunk 1]
```diff
@@ -123,16 +123,17 @@ The following values are defined for the status of a session:
...
-| `mfa`    | Multiple factor authentication    | (This value is present whenever any MFA factor verification is performed) |
-|--------- +-----------------------------------+---------------------------------------------------------------------------|
+|----------+------------------------------------------------------+---------------------------------------------------------------------------|
+| Value    | Description                                          | Example                                                                   |
+| -------- | -----------------------------------------------------|---------------------------------------------------------------------------|
+| `pwd`    | Password authentication                              | Standard password-based login                                             |
+| `swk`    | Proof-of-possession (PoP) of a software-secured key. | Okta Verify with Push                                                     |
+| `hwk`    | Proof-of-possession (PoP) of a hardware-secured key. | Duo Security factor                                                       |
```
Comment (tthompson-okta) [inline _source/docs/api/resources/sessions.md]: Needs to mention the Yubikey factor.

Comment (larsjohansen-okta) [inline _source/docs/api/resources/sessions.md]: My bad, mixed up Duo with Yubikey..

**Comment by tthompson-okta** [comment]
🚢

**Pull request merged**